src/editor/components/EditorPanel.tsx

```diff
@@ -39,7 +39,9 @@ export const EditorPanel = () => {
         <Tab label='Preview' />
       </Tabs>
       <TabContent index={0} currentIndex={selectedTab}>
-        <Editor />
+        <div style={{ padding: 10 }}>
```

Comment:
can't we add the style to the TabContent instead ?

Reply:
I don't think we should pad tab elements but probably the whole pane at once. Also I would much prefer using `makeStyle`, `useStyle` and using theme support for this.

Reply:
We need to pad something, and if we decide to pad the TabContent we should do so in the TabContent class not in a separate div

Reply:
I think this padding was meant for the editor to not be so close to the splitters. However the preview is still so close. So I would pad the whole panel.
Regarding styling, yes if possible I would also avoid creating divs only for styling if another approach is possible.